Neos.Neos/Resources/Public/JavaScript/index.js

```diff
@@ -1,19 +1,50 @@
-import {DropDownMenu, MenuPanel} from './Components/TopBar'
-import Tree from './Components/Tree'
+import jQuery from "jquery";
 
-const dropDownMenuElements = document.querySelectorAll('.neos-user-menu');
-dropDownMenuElements.forEach(dropDownElement => {
+import { DropDownMenu, MenuPanel } from "./Components/TopBar";
+import Tree from "./Components/Tree";
+import Modal from "./Components/Modal";
+import { Configuration, Notification, Localization, Tools } from "./Services";
+import { cachedFetch } from "./Services/ResourceCache";
+import { isNil } from "./Helper";
+
+// export jQuery globally
+window.jQuery = jQuery;
+window.$ = jQuery;
+
+// init API's
+Tools.init();
+Configuration.init();
+Notification.init();
+
+// preload vieSchema
+const vieSchema = cachedFetch(Configuration.get("VieSchemaUri"));
+cachedFetch(Configuration.get("XliffUri")).then((xliffData) => {
+	Localization.init(xliffData);
+});
+
+const dropDownMenuElements = document.querySelectorAll(".neos-user-menu");
+dropDownMenuElements.forEach((dropDownElement) => {
 	new DropDownMenu(dropDownElement);
 });
 
-const menuPanelElements = document.querySelectorAll('.neos-menu');
-menuPanelElements.forEach(panelElement => {
+const menuPanelElements = document.querySelectorAll(".neos-menu");
+menuPanelElements.forEach((panelElement) => {
 	new MenuPanel(panelElement);
 });
 
-const treeElements = document.querySelectorAll('.neos-tree-container');
-treeElements.forEach(treeElement => {
+const treeElements = document.querySelectorAll(".neos-tree-container");
+treeElements.forEach((treeElement) => {
 	new Tree(treeElement);
 });
 
-
+const modalTrigger = Array.from(
```

Comment:
Why do you always use `Array.from(someSelector)` and not just iterate over the result?

Reply:
That is more a common style of mine... the problem is that you get a NodeList returned. For the general forEach that is no issue. But when you want to use filter or map it does not work. Because NodeList is array like but not an Array. And therefore I mostly use Array.from to have an true array. So it is more or less laziness to change the stuff when I come to a place where I want to use map, filter and so on.